src/mount/mount-tool.c

```diff
@@ -77,7 +78,9 @@ static void polkit_agent_open_if_enabled(void) {
 }
 
 static void help(void) {
-        printf("%s [OPTIONS...] WHAT [WHERE]\n\n"
+        printf("systemd-mount [OPTIONS...] WHAT [WHERE]\n"
+               "systemd-mount [OPTIONS...] --list\n"
+               "systemd-umount [OPTIONS...] WHAT|WHERE...\n\n"
```

Comment:
hmm, i'd probably stick to "systemd-mount --umount" for the third item too, to make this more uniform.